modules/BookCollection.js

```diff
@@ -0,0 +1,45 @@
+import Book from './Book.js';
+
+export default class BookCollection {
+  constructor() {
+    this.books = JSON.parse(localStorage.getItem('books')) || [];
+  }
+
+  // Using classes instead of objects for add book function.
+  addBook = (title, author) => {
+    const book = new Book(title, author);
+
+    this.books.push(book);
+    this.saveToLocalStorage();
+  }
+
+  removeBook = (title) => {
+    this.books = this.books.filter((book) => book.title !== title);
+    this.saveToLocalStorage();
+  }
```

Comment:
- [ ] 👍 As I mentioned in my previous review, our Booklist implementation with the book title and author properties is looking good, and the removeBook function is working well. However, there is an issue worth addressing:
Currently, if there are multiple books with the same title but different authors, the removeBook function deletes all instances of that book title, regardless of the author. To prevent this behavior and ensure better book identification, I recommend adding an additional attribute, such as an "id" property, to uniquely identify each book in the book list. By assigning a unique "id" to each book, you can accurately identify and remove the specific book that needs to be deleted, avoiding the deletion of other books with the same title but different authors.

[Awesome-Books-with-ES6-Modules (1).webm](https://github.com/Omwomo/Awesome-books-with-ES6/assets/55840999/95da38c3-3570-4618-866b-b74b0f24e395)

Incorporating this approach will improve the reliability and precision of your Booklist functionality. Keep up the good work, and I hope this recommendation proves useful! 📚🔍🚀